src/main.rs

```diff
@@ -44,8 +45,24 @@ pub fn main() {
 
     match builder_from_flags(bind_args.into_iter()) {
         Ok((builder, output)) => {
-            let mut bindings = builder.generate()
-                .expect("Unable to generate bindings");
+
+            let builder_result =
+                panic::catch_unwind(||
+                    builder.generate().expect("Unable to generate bindings")
+                );
+
+            if builder_result.is_err() {
```

Comment:
Also, could we put this after a false-by-default option until we actually write those links? :)